lsp-ui-doc.el

```diff
@@ -629,7 +630,12 @@ FN is the function to call on click."
         (goto-char next)
         (setq bolp (bolp)
               before (char-before))
-        (kill-line 1)
+        (delete-region (point) (save-excursion
+                                 (forward-visible-line 1)
+                                 (if (eobp)
+                                     (signal 'end-of-buffer nil))
+                                 (end-of-visible-line)
```

Comment:
```suggestion
```

This `delete-region` is deleting too much:
![image](https://user-images.githubusercontent.com/5273820/93670285-e9bd6f80-facc-11ea-9e1c-327a3fad4f4f.png)

Without the `end-of-visible-line`:
![image](https://user-images.githubusercontent.com/5273820/93670268-c5619300-facc-11ea-99d9-90ef8305de72.png)

Reply:
Ah crap that is pretty poor on my part.  Thanks for checking, I'm going to add some tests for these 2 functions i think